### PR TITLE
Guest user information Added

### PR DESCRIPTION
**Pull request by ojrb**
This PR solved the issue #485 I add the reference and the information to Add Guest users to groups on the Tenant, please review and merge!

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/microsoft-365-guest-settings.md [hunk 1]
```diff
@@ -41,6 +41,12 @@ Azure Active Directory is the directory service used by  Microsoft 365. The Azur
 
 These settings affect how users are invited to the directory. They do not affect sharing with guests who are already in the directory.
 
+Refer to [Azure Active Directory cmdlets for configuring group settings](https://docs.microsoft.com/azure/active-directory/users-groups-roles/groups-settings-cmdlets), in order to know how group settings for guest users can be controlled at Tenant level.
```
Comment (JohanFreelancer9) [inline Enterprise/microsoft-365-guest-settings.md]:
```suggestion
Refer to [Azure Active Directory cmdlets for configuring group settings](https://docs.microsoft.com/azure/active-directory/users-groups-roles/groups-settings-cmdlets) in order to learn how group settings for guest users can be controlled at Tenant level.  
```

File: Enterprise/microsoft-365-guest-settings.md [hunk 2]
```diff
@@ -41,6 +41,12 @@ Azure Active Directory is the directory service used by  Microsoft 365. The Azur
 
 These settings affect how users are invited to the directory. They do not affect sharing with guests who are already in the directory.
 
+Refer to [Azure Active Directory cmdlets for configuring group settings](https://docs.microsoft.com/azure/active-directory/users-groups-roles/groups-settings-cmdlets), in order to know how group settings for guest users can be controlled at Tenant level.  
+
+If you want that group owners can add guest users as a member of a group, use the option **AllowToAddGuests**.
```
Comment (JohanFreelancer9) [inline Enterprise/microsoft-365-guest-settings.md]:
```suggestion
If you want group owners to be able to add guest users as members of a group, use the option **AllowToAddGuests**.
```

File: Enterprise/microsoft-365-guest-settings.md [hunk 3]
```diff
@@ -41,6 +41,12 @@ Azure Active Directory is the directory service used by  Microsoft 365. The Azur
...
+
+If you want that guest users members of a group can access the group information, use the option **AllowGuestsToAccessGroups**.
```
Comment (JohanFreelancer9) [inline Enterprise/microsoft-365-guest-settings.md]:
```suggestion
If you want guest user members of a group to be able to access group information, use the option **AllowGuestsToAccessGroups**. 
```

**Comment by mypil** [comment]
@ojrb  - please merge the commit on the copy/edit review made by JohanFreelancer9  or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by ojrb** [comment]
All suggestions committed ! Please review and merge !

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #485. 

Thank you.

**Comment by MikePlumleyMSFT** [comment]
This information is already covered in the Office 365 Groups section of the article.

**Pull request closed without merging**